meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -27,6 +27,30 @@ FILES:${PN} += " \
 # It is used a lot in the do_install, so variable will be handy
 CFG_FILE="${D}${sysconfdir}/xen/domu.cfg"
 
+# This function is used to determine `dtdev` block for GSX.
+# Pay attention, that single quote char ' have to be used
+# inside `dtdev = []` construction because we will use
+# linux's echo command to put multiline string into domu.cfg
+python () {
+    if d.getVar("XT_DOMU_CONFIG_NAME", True) == "domu-generic-h3-4x2g.cfg":
```

Comment:
This looks fragile.

I'd prefer to see use of flags here instead of inline python. Something like

```GSX_DTDEV[h3] = \
    '/soc/gsx_pv0_domu',
    '/soc/gsx_pv1_domu',
    '/soc/gsx_pv2_domu',
    '/soc/gsx_pv3_domu',
```

and then some logic (maybe even inline python, okay) that choose either m3 or h3